sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/cosmos/CosmosAutoConfiguration.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.spring.autoconfigure.cosmos;
```

Comment:
We didn't change this package name in this PR?

Reply:
Jack Wu will change it later.